ftp/src/test/scala/akka/stream/alpakka/ftp/CommonFtpStageSpec.scala

```diff
@@ -278,56 +286,60 @@ trait CommonFtpStageSpec extends BaseSpec with Eventually {
       extraWaitForStageShutdown()
     }
 
-    "fail and report the exception in the result status if connection fails" in { // TODO Fails too often on Travis: assertAllStagesStopped {
+    "fail and report the exception in the result status if connection fails" ignore { // TODO Fails too often on Travis: assertAllStagesStopped {
       def waitForUploadToStart(fileName: String) =
         eventually {
-          noException should be thrownBy getFtpFileContents(FtpBaseSupport.FTP_ROOT_DIR, fileName)
-          getFtpFileContents(FtpBaseSupport.FTP_ROOT_DIR, fileName).length shouldBe >(0)
+          noException should be thrownBy getFtpFileContents(fileName)
+          getFtpFileContents(fileName).length shouldBe >(0)
         }
 
       val fileName = "sample_io_connection"
       val infiniteSource = Source.repeat(ByteString(0x00))
 
       val future = infiniteSource.runWith(storeToPath(s"/$fileName", append = false))
       waitForUploadToStart(fileName)
-      stopServer()
+      // stopServer()
```

Comment:
You may drop these now. Yay.

Reply:
You mean drop the whole "fail and report the exception in the result status if connection fails" test? It is currently ignored and I thought you might want to implement it another way.

Reply:
Aha, I didn't notice it is ignored. I meant the `stopServer()` / `startServer()` comments. Leave it as it is.